examples/tutorial/vertical_federated_learning/cifar10-splitnn/job_configs/cifar10_splitnn/meta.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "cifar10_splitnn",
```

Comment:
I notice in cifar10 examples, we use "underscore" for all jobs.
VS 
other examples I am using "-".

Let us decide 1 pattern later, and I will change all this to a unified form.

Reply:
Maybe not essential but sure.